build-system/check-package-manager.js

```diff
@@ -279,16 +320,16 @@ function main() {
   }
   ensureYarn();
   return checkNodeVersion().then(() => {
-    checkGlobalGulp();
+    runGulpChecks();
     checkYarnVersion();
-    if (!process.env.TRAVIS && updatesNeeded.length > 0) {
+    if (!process.env.TRAVIS && updatesNeeded.size > 0) {
       console.log(
-        yellow('\nWARNING: Detected missing updates for'),
-        cyan(updatesNeeded.join(', '))
+        yellow('\nWARNING: Detected problems with'),
+        cyan(Array.from(updatesNeeded).join(', '))
       );
       console.log(
         yellow('⤷ Continuing install in'),
-        cyan('5'),
+        cyan('10'),
```

Comment:
can we move this value up to a const var, then just do `value * 1000` in the timeout

Reply:
Done in f25bed7e6899d66d52954843a534cdc5312143f8